iothub/service/samples/getting started/EdgeDeploymentSample/EdgeDeploymentSample.cs

```diff
@@ -143,7 +143,7 @@ private static IEnumerable<Device> GenerateEdgeDevices(string deviceIdPrefix, in
 
         private Task<BulkRegistryOperationResult> CreateEdgeDevicesAsync(IEnumerable<Device> edgeDevices)
         {
-            return _registryManager.AddDevices2Async(edgeDevices);
+            return _serviceClient.Devices.CreateAsync(edgeDevices);
         }
```

Comment:
Seems silly for a sample to have a helper method that simply delegates to another call. I'd simplify this.